addons/sale_management/static/src/js/sale_management.js

```diff
@@ -66,19 +67,20 @@ if (!$('.o_portal_sale_sidebar').length) {
          * when the quantity of a product has changed
          *
          * @private
+         * @param {Int} order_id
```

Comment:
I think we usually write it `{integer}` (and to be exact there is only `Number` in JS, so we should use that instead and specify in the description that integer is expected).